app/markets/[id]/MarketDetails.tsx

```diff
@@ -78,21 +78,30 @@ const LoadingMarketDetails = () => (
       <div className="size-8 rounded-8 bg-outline-low-em animate-pulse"></div>
       <div className="w-14 h-8 rounded-8 bg-outline-low-em animate-pulse"></div>
     </div>
-    <div className="bg-surface-surface-0 w-full max-w-[464px] h-[544px] rounded-16 border border-outline-base-em">
+    <section className="bg-surface-surface-0 w-full max-w-[464px] rounded-16 border border-outline-base-em">
       <div className="p-5 space-y-4">
         <div className="flex justify-between items-center">
           <div className="w-20 h-7 rounded-8 bg-outline-low-em animate-pulse" />
           <div className="w-28 h-4 rounded-8 bg-outline-low-em animate-pulse" />
         </div>
         <div className="flex space-x-4">
           <div className="size-20 rounded-8 flex-shrink-0 bg-outline-low-em animate-pulse" />
-          <div className="w-full h-20 rounded-8 bg-outline-low-em animate-pulse" />
+          <div className="w-full h-28 rounded-8 bg-outline-low-em animate-pulse" />
         </div>
-        <div className="!mt-7 w-full h-7 rounded-8 bg-outline-low-em animate-pulse" />
+        <div className="!mt-7 w-full h-10 rounded-8 bg-outline-low-em animate-pulse" />
       </div>
-      <div className="p-2 w-full h-[348px]">
+      <div className="p-2 w-full h-[112px]">
```

Comment:
```suggestion
      <div className="p-2 w-full h-28">
```

Reply:
h-28 doesn't correspond to 112px since our base font-size is 14px and sizes are calculated in rem. Default size is 16px, so h-28 will be a little less than 112px. I'm ok with either since this is a loading skeleton.

Reply:
Yeah, I think we can stay with the `px`.

But changing the base font breaks a lot of tailwind conventions, we probably should be using the normal base 16px and set the default size on the `body` tag as `text-sm`.

what do you think?

Reply:
That sounds better, let's do that.

Reply:
Created a task. I'll be fixed.